Training_Center_Task_11_Selenium_Tests/ConstantsTests.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Training_Center_Task_11_Selenium_Tests
+{
+    public class ConstantsTests
+    {
+        public const int WebDriverWaitTime = 10;
+        public const string WebDriverPath = @"C:\Users\IvoIgov\source\repos\Training_Center_Task_11\Training_Center_Task_11_Selenium_Tests\bin\Debug\net6.0\";
```

Comment:
please use driver from nuget package

Reply:
Fixed this entirely, removed the whole class